mobile/lib/screens/manage-vehicles/add_vehicle_screen.dart

```diff
@@ -35,227 +35,334 @@ class AddVehicleScreenState extends State<AddVehicleScreen> {
     super.dispose();
   }
 
+  String _dropdownValue = "Car";
+  List<String> dropDownOptions = [
+    "Car",
+    "Van",
+    "Bike",
+  ];
+
+  void dropdownCallback(String? selectedValue) {
+    if (selectedValue is String) {
+      setState(() {
+        _dropdownValue = selectedValue;
+      });
+    }
+  }
+
   @override
   Widget build(BuildContext context) {
     final Size size = MediaQuery.of(context).size;
     const double padding = 16;
     const sidePadding = EdgeInsets.symmetric(horizontal: padding);
-    return Scaffold(
-      body: SizedBox(
-        width: size.width,
-        height: size.height,
-        child: Padding(
-          padding: sidePadding,
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              addVerticalSpace(44),
-              const Align(
-                  alignment: Alignment.topLeft,
-                  child: Icon(
-                    EvaIcons.arrowBackOutline,
-                    color: Colors.black,
-                  )),
-              addVerticalSpace(42),
-              const Text(
-                'Enter vehicle information',
-                style: BlipFonts.title,
-              ),
-              addVerticalSpace(36),
-              Align(
-                child: Image.asset('assets/images/car.png', height: 137),
-              ),
-              Form(
-                key: _formKey,
-                child: Column(
-                  children: [
-                    Row(
+
+    return SafeArea(
+      child: Scaffold(
+        body: SingleChildScrollView(
+          child: SizedBox(
+            width: size.width,
+            height: size.height,
+            child: Padding(
+              padding: sidePadding,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  addVerticalSpace(12),
+                  Align(
```

Comment:
Let's remove the back button here and put it in the app bar like this
![image](https://user-images.githubusercontent.com/34070216/190636271-ffdc14db-dd71-4c9d-886f-de725a28e240.png)